src/index.js

```diff
@@ -0,0 +1,62 @@
+import './style.css';
+import * as Store from './modules/store.js';
+import UIDisplay from './modules/showTask.js';
+import taskArr from './modules/toDo.js';
```

Comment:
- [ ] In this `toDo` file, we have no hardcoded task, because of that unable to see any tasks. Either add some task or you can use the `ToDoList.js` file. As I can see there is some task written there. Please update it.